app/cmd/computer_launch.go

```diff
@@ -213,7 +215,14 @@ jcli agent launch agent-name --type jnlp`,
 		}
 
 		var f *os.File
-		if f, err = ioutil.TempFile("/tmp", "agent.jar"); err == nil {
+		var tmpPath = "/tmp"
```

Comment:
IMO, we could use https://pkg.go.dev/io/ioutil@go1.16#TempDir to create a temporary directory here. And based on the temporary directory, we can create any files inside it.